patches/react-native+0.73.4+006+Codegen.patch

```diff
@@ -0,0 +1,21 @@
+diff --git a/node_modules/react-native/scripts/cocoapods/new_architecture.rb b/node_modules/react-native/scripts/cocoapods/new_architecture.rb
+index ba75b01..e5eaee8 100644
```

Comment:
@WoLewicki , I failed to find in the listed PR the modifications of this patch, can you point me out the PR? 🙏

Reply:
I [pinged](https://expensify.slack.com/archives/C04878MDF34/p1713719043648679) @WoLewicki in Slack

Reply:
I believe that https://github.com/facebook/react-native/pull/41715 did this on top of some other things